### PR TITLE
Addition of Plum Gathering

### PR DESCRIPTION
**Pull request by KasperOnPC**
First Contribution to get used to the process/learning.

Context from: 
From: https://github.com/EveronLife/EveronLife/discussions/49

Apologies in advance if there are any issues with this. Play tested and added Plum Tree to Debug Environment for testing.

### CODE REVIEW

**Review by alexhosier — APPROVED** [review]
All tested, working in-game, changes ready to be pulled to main.

**Comment by Arkensor** [comment]
Hey @KasperOnPC, thank you for your contribution. Before looking at anything else, I must note that the model can not be used. The license (https://blog.turbosquid.com/turbosquid-3d-model-license/#Games-Usage) does not permit free open access and can only be bundled with a closed game binary. To my knowledge, this is not compatible with the MIT license.

The model needs to be swapped out. You can find license complatible models (e.g. CC-BY) here: https://sketchfab.com/search?features=downloadable&licenses=322a749bcfa841b29dff1e8a1bb74b0b&type=models

**Comment by KasperOnPC** [comment]
That's my bad @Arkensor, thanks for the detailed feedback. It looks like there are some plum models under CC-BY, I will switch it out tomorrow.

Would love any other feedback and appreciate how small a contribution this is. Thanks for all you do.

**Comment by KasperOnPC** [comment]
Updated with a file with the correct license @Arkensor. I have also ensured the texture and 3D file are super small, 1/100th of the size of the apple.

File: src/Assets/Items/Food/Plum/Plum.xob.meta [hunk 1]
```diff
@@ -0,0 +1,31 @@
+MetaFileClass {
+ Name "{1768DEF76C703B91}Assets/Items/Food/Plum/Plum_01.xob"
```
Comment (Arkensor) [inline src/Assets/Items/Food/Plum/Plum.xob.meta:2]: This still has the old path in it.

File: src/Assets/Items/Food/Plum/data/Plum.emat.meta [hunk 2]
```diff
@@ -0,0 +1,15 @@
+MetaFileClass {
+ Name "{5DCEE3E3833E90FA}Assets/Items/Food/Plum/data/Plum_01.emat"
```
Comment (Arkensor) [inline src/Assets/Items/Food/Plum/data/Plum.emat.meta:2]: This still has 01 in the path

File: src/Prefabs/Items/Food/Plum.et [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    }
+    CustomAttributes {
+     ItemActionAnimAttributes "{597CB6B0AB826EDE}" {
+      ActionType Bandage
+     }
+    }
+   }
+  }
+  MeshObject "{597CB6B0AB826ED8}" {
+   Object "{1768DEF76C703B91}Assets/Items/Food/Plum/Plum_01.xob"
```
Comment (Arkensor) [inline src/Prefabs/Items/Food/Plum.et:42]: Links to the old xob file

**Review by Arkensor — CHANGES_REQUESTED** [review]
- Remove `src/Assets/Items/Food/Plum/Plum.fbx` that should only be in `assets`
- Remove `src/Assets/Items/Food/Plum/Plum.txo` this file is not needed I think. According to the docs, it is just the source of the `xob` binary file. Will be ignored in packing anyway. 
- Some of the paths do not seem to be refreshed and still include `_01`. Not sure how to refresh them. Maybe Rightclick and set no value on the attributes and then select the source again.

File: src/Assets/Items/Food/Plum/data/Plum.emat.meta (hunk 2 above)
Comment (Arkensor) [reply src/Assets/Items/Food/Plum/data/Plum.emat.meta:2]: Still not resolved

**Review by Arkensor — CHANGES_REQUESTED** [review]
- `src/Assets/Items/Food/Plum/data/Plum.emat.meta` still references `Plum_01.emat`
- Please do not modify other files unrelated to your pull request. Like you did with the apple. Revert that commit for your branch, please.

**Pull request merged**